apps/user-office-frontend/src/components/user/PeopleTable.tsx

```diff
@@ -337,37 +350,33 @@ const PeopleTable: React.FC<PeopleTableProps> = (props) => {
     orderDirection: 'desc' | 'asc'
   ) => {
     if (
-      columns[orderedColumnId] &&
+      localColumns[orderedColumnId] &&
```

Comment:
Would be great if you can really breifly explain the reson behind why changed the naming to localColumns

Reply:
After removing the `props` from `props.columns` and we had `columns` defined in the same file there was a collision in the naming. I had to change the local ones with `localColumns` and keep the property columns with just `columns` name